README.md

```
@@ -27,16 +29,20 @@ KiraOS 是 Kira 的 OS 级插件，整合了两大核心能力：
   - [创建自定义技能](#创建自定义技能)
   - [manifest.json 规范](#manifestjson-规范)
   - [instruction.md 规范](#instructionmd-规范)
-  - [完整示例：塔罗牌占卜](#完整示例塔罗牌占卜)
+  - [示例技能](#示例技能)
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)